images/fcos-bastion-image/root/usr/bin/dhcp-init.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -x
+
+mkdir -p /var/opt/{ignitions,html}
+cp ${LOCAL_PATH}/dhcp/ignitions/* /var/opt/ignitions/
+
+ign=/var/opt/ignitions
+mkdir -p ${PODMAN_DNSMASQ_BASEDIR}/{etc,tftpboot,misc}
+#/usr/bin/chown -R 65534:65534 ${PODMAN_DNSMASQ_BASEDIR}/misc
+cp ${LOCAL_PATH}/dhcp/bastion_dnsmasq.conf ${PODMAN_DNSMASQ_BASEDIR}/etc/
+cp ${LOCAL_PATH}/dhcp/tftpboot/grub.cfg ${TFTP_DIR}
+cp ${LOCAL_PATH}/ignitions/* ${ign}
+
+# create ignition files
+export password_hash=$(<"${ign}/password_hashes")
+
+pdm_opts="--rm --interactive --security-opt label=disable -v ${ign}:/ign -v /var/opt/html:/output -w /ign"
```

Comment:
use an array here and then expand it with `${pdm_opts[*]}`

Reply:
done